src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -15,18 +16,25 @@
 
 (trapperkeeper/defservice jruby-puppet-pooled-service
                           jruby/JRubyPuppetService
-                          [[:ConfigService get-config]
-                           [:ShutdownService shutdown-on-error]
-                           [:PuppetProfilerService get-profiler]
-                           [:PoolManagerService create-pool]]
+                          {:required [[:ConfigService get-config]
+                                      [:ShutdownService shutdown-on-error]
+                                      [:PuppetProfilerService get-profiler]
+                                      [:PoolManagerService create-pool]]
+                           :optional [MetricsService]}
```

Comment:
Right now we only have the ability to enable or disable the entire metrics system right? Would adding a setting just for the http metrics be weird?